### PR TITLE
Fix master config paths

### PR DESCRIPTION
**Pull request by michaelgugino**

### CODE REVIEW

File: roles/openshift_control_plane/tasks/main.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
 - name: Create the ldap ca file if needed
   copy:
-    dest: "{{ item.ca if 'ca' in item and '/' in item.ca else openshift_master_config_dir ~ '/' ~ item.ca | default('ldap_ca.crt') }}"
+    dest: "{{ item.ca if 'ca' in item and '/' in item.ca else '/etc/origin/master/' ~ item.ca | default('ldap_ca.crt') }}"
```
Comment (michaelgugino) [inline roles/openshift_control_plane/tasks/main.yml]: Need to do something to migrate this similar to htpasswd.

File: roles/openshift_control_plane/tasks/main.yml [hunk 2]
```diff
@@ -57,7 +57,7 @@
 
 - name: Create the openid ca file if needed
   copy:
-    dest: "{{ item.ca if 'ca' in item and '/' in item.ca else openshift_master_config_dir ~ '/' ~ item.ca | default('openid_ca.crt') }}"
+    dest: "{{ item.ca if 'ca' in item and '/' in item.ca else '/etc/origin/master/' ~ item.ca | default('openid_ca.crt') }}"
```
Comment (michaelgugino) [inline roles/openshift_control_plane/tasks/main.yml]: Need to do something to migrate this similar to htpasswd.

File: roles/openshift_control_plane/tasks/main.yml [hunk 3]
```diff
@@ -69,7 +69,7 @@
 
 - name: Create the request header ca file if needed
   copy:
-    dest: "{{ item.clientCA if 'clientCA' in item and '/' in item.clientCA else openshift_master_config_dir ~ '/' ~ item.clientCA | default('request_header_ca.crt') }}"
+    dest: "{{ item.clientCA if 'clientCA' in item and '/' in item.clientCA else '/etc/origin/master/' ~ item.clientCA | default('request_header_ca.crt') }}"
```
Comment (michaelgugino) [inline roles/openshift_control_plane/tasks/main.yml]: Need to do something to migrate this similar to htpasswd.

**Comment by sdodson** [comment]
@smarterclayton We're doubling down on /etc/origin/master here by removing the ability to configure the config path, sound good?

**Comment by vrutkovs** [comment]
bot, retest this please

**Comment by sdodson** [comment]
bot, retest this please

**Pull request merged**